packages/popover/src/Popover.tsx

```diff
@@ -93,24 +93,29 @@ function Popover({
 
   const viewportSize = useViewportSize();
 
+  // We already recalculate the position when the element becomes active,
+  // So it's safe for us to disable the mutation observers when the element isn't active.
+  const observeMutations = adjustOnMutation && active;
+
   const lastTimeRefElMutated = useMutationObserver(
     referenceElement,
     mutationOptions,
-    () => Date.now(),
-    adjustOnMutation,
+    useCallback(() => Date.now(), []),
```

Comment:
does this work `useCallback(Date.now, [])`

Reply:
Makes sense!